csaf_2.0/test/validator/testcases_json_schema.json

```diff
@@ -79,10 +79,20 @@
     }
   },
   "required": [
+    "$schema",
```

Comment:
In general this is ok for me, since this is also a test file, one just needs to be aware of that specifying the `$schema` as a property is not really "recommended" or specified by the standard. 

Although it is used by many editors (see VSCode https://code.visualstudio.com/docs/languages/json#_mapping-in-the-json) there is no official statement from the specification on that (see https://github.com/orgs/json-schema-org/discussions/473). In the fact the discussion there did not end with any conclusion at all.

I will conduct a small test, whether this actually works at least with the 2 JSON schema generator libraries that we use

Reply:
@oxisto Could you please double check that the JSON schema spec does not prohibit the use of `$schema` in a JSON instance and as defined in the JSON schema. (This might have an impact on #616.)

At the moment, I don't see another way to make sure the location of the schema is known other than to declare which schema should be used to validate ...

Reply:
Done in https://github.com/oasis-tcs/csaf/pull/828#issuecomment-2501283070